src/pages/DACDatasets.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState, useCallback } from 'react';
+import { button, div, h, span } from 'react-hyperscript-helpers';
+import {DataSet, User} from '../libs/ajax';
+import {Styles} from '../libs/theme';
+import {img} from 'react-hyperscript-helpers';
+import lockIcon from '../images/lock-icon.png';
+import SearchBar from '../components/SearchBar';
+import {DACDatasetTable, DACDatasetTableColumnOptions} from '../components/dac_dataset_table/DACDatasetTable';
+import {getSearchFilterFunctions, Notifications, searchOnFilteredList} from '../libs/utils';
+import {useRef} from 'react';
+import { consoleTypes } from '../components/dac_dataset_table/DACDatasetTableCellData';
+import '../components/dac_dataset_table/dac_dataset_table.css';
+
+export default function DACDatasets(props) {
```

Comment:
should there be a `DataSubmitterDatasets` component as well? would they show different thigns?